src/components/Logs.tsx

```diff
@@ -1,30 +1,62 @@
-import { useState } from 'react';
+import Editor from '@monaco-editor/react';
+import { useTheme } from '@mui/material';
+import { parse } from 'ansicolor';
+import * as monacoEditor from 'monaco-editor/esm/vs/editor/editor.api';
+import { useRef, useState } from 'react';
 import { useIntl } from 'react-intl';
-import { LazyLog } from 'react-lazylog';
 import { useInterval } from 'react-use';
 import { DEFAULT_POLLING_INTERVAL, RPCS } from 'services/supabase';
 import { useClient } from 'supabase-swr';
 
 interface Props {
     token: string | null;
+    height?: number;
     defaultMessage?: string;
 }
 
 const NEW_LINE = '\r\n';
 
-function Logs({ token, defaultMessage }: Props) {
+const generateLogLine = (logData: any) => {
+    const parsedText = parse(logData.log_line);
+    const mergedText = parsedText.spans
+        .map((span) => {
+            return span.text;
+        })
+        .join('');
+
+    return mergedText;
+};
```

Comment:
This will need a lot more work. The ansi parser here returns and tells us how to style specific pieces of text. We eventually need to get that stuff passed/formatted for Monaco.